.github/mergify.yml

```diff
@@ -36,10 +38,12 @@ pull_request_rules:
 
   - name: Update labels when PR is unblocked
     conditions:
+      - -closed
+      - -draft
       - label=waiting-on-author
       - -conflict
-      - check-failure!=test-suite-success
-      - check-failure!=local-testnet-success
+      - check-skipped!=test-suite-success
+      - check-skipped!=local-testnet-success
```

Comment:
I think you need all 4 of these, otherwise we have the same problem in reverse don't we. If the checks fail, we add waiting-on-author, but then this job adds ready-for-review immediately after and then we loop.

Reply:
Added